check_haproxy.rb

```diff
@@ -85,16 +91,28 @@
   exit UNKNOWN
 end
 
+
+tries = 2
+
 begin
   f = open(options.url, :http_basic_authentication => [options.user, options.password])
 rescue OpenURI::HTTPError => e
   puts "ERROR: #{e.message}"
-  exit UNKNOWN
+  exit CRITICAL
```

Comment:
I check the HAProxies locally, not remotely. So the HAProxy should run all the time.

I understand if you check HAProxy remotely, then you don't have to be bothered with some short networks issues.

I will create the option for that.